### ISSUE TITLE
Preparation script not working

### ISSUE THREAD

**Issue by outcatcher** [opentelekomcloud-infra/csm-sandbox#5]
`prepare_cloud_config.sh` is not working: 

following error is shown:
```log
clouds.yml and secure.yml should exist in current directory.
Directory /etc/openstack not exists.
mkdir: cannot create directory ‘/etc/openstack’: Permission denied
cp: target ‘/etc/openstack’ is not a directory
```

**Comment by outcatcher** [comment]
Also, message `clouds.yml and secure.yml should exist in current directory.` is shown even if those files exist — this can be misleading

**Comment by outcatcher** [comment]
Fixed in #8
